tensorflow_hub/tools/make_image_classifier/make_image_classifier.py

```diff
@@ -153,6 +159,18 @@ def _assert_accuracy(train_result, assert_accuracy_at_least):
   else:
     raise AssertionError("ACCURACY FAILED:", accuracy_message)
 
+def _set_gpu_memory_growth():
+  gpus = tf.config.experimental.list_physical_devices('GPU')
+  if gpus:
+    try:
+      # Currently, memory growth needs to be the same across GPUs
+      for gpu in gpus:
+        tf.config.experimental.set_memory_growth(gpu, True)
+      logical_gpus = tf.config.experimental.list_logical_devices('GPU')
```

Comment:
These two lines are unrelated to set_memory_growth. Maybe you can just delete them. If you insist on having a message on the number of devices, make it independent of the set_memory_growth functionality.